forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -461,7 +461,7 @@ public void createCheck(Check check) {
     @Override
     public void updateCheck(Check check) {
         var completedQuery = JSON.object();
-        completedQuery.put("name", check.name());
+        completedQuery.put("name", check.name().equals("jcheck") ? check.name() + repository().name() + id() : check.name());
```

Comment:
I don't understand this change. Why would we need that? I don't think GitHubPullRequest should be changing check names.

Reply:
As you said in the comment above "The check is created in CheckRun, so the name needs to be changed there too.", I didn't change the check name when it is created, so I changed it here. It's my bad, I will fix it.